awx/settings/local_settings.py.docker_compose

```diff
@@ -238,3 +243,8 @@ TEST_OPENSTACK_PROJECT = ''
 # Azure credentials.
 TEST_AZURE_USERNAME = ''
 TEST_AZURE_KEY_DATA = ''
+
+BROADCAST_WEBSOCKET_SECRET = '🤖starscream🤖'
```

Comment:
Is this something people can specify/override in the installer?

Reply:
Is this just for docker-compose?

Reply:
I _could_ expose this in the installer inventory. However, a user shouldn't need to know/care nor change it unless it's compromised. Unlike other secrets that we have, this could be randomly generated by our installer on each run.